### ISSUE TITLE
[feature request] dahd lin to exp window

### ISSUE THREAD

**Issue by Nowhk** [nysthi/nysthi#159]
If I increse the lin to exp knob, it "reduce" the window of the envelope. 

It should just change the "tension" of each rise/fall, not decrease the whole time of each.

**Comment by nysthi** [comment]
you are asking for constant time exponential skewing: 👍 
ok I'll take a look when I'll have time

**Comment by Nowhk** [comment]
Yep, constant time between linear/exp. Also, linear its seems not proper "linear" (I always see a bit of exp).
Also: what about introduce log?
Thanks
